### DISCUSSION TITLE
When to use 'worklet' ?

### DISCUSSION THREAD

**Discussion by mtlaso** [software-mansion/react-native-gesture-handler#2113] [Q&A]
Hello,

I am very new to using `react-native-gesture-handler`. One thing I can't seem to grasp is when and where the 'worklet' keyword is needed. Is there a clear explanation on the docs ? Thank you!

**Comment by j-piasecki** [comment]
It's not in the docs (although it should be, will fix it), [here's the answer](https://github.com/software-mansion/react-native-gesture-handler/issues/1821#issuecomment-1057005397).
